Sources/_SwiftFormatTestSupport/DiagnosingTestCase.swift

```diff
@@ -213,54 +213,45 @@ open class DiagnosingTestCase: XCTestCase {
     file: StaticString = #file,
     line: UInt = #line
   ) {
-    // Use `CollectionDifference` on supported platforms to get `diff`-like line-based output. On
-    // older platforms, fall back to simple string comparison.
-    if #available(macOS 10.15, *) {
```

Comment:
I removed what seemed to be unnecessary OS conditionals. If the GitHub Actions CI passes, can we assume it works fine on Linux and Windows?

Reply:
Yeah, as long as all the CI comes back green, that should be enough verification that the versions of Foundation we care about have the necessary APIs. (They should anyway, because of the `*` in the `#available` condition.)

Reply:
Ah, since there is a `*` it seems that it was already working on Windows and Linux 😅
Thank you for letting me know.